### PR TITLE
fix(spike protection): mention all applicable categories

### PR DESCRIPTION
**Pull request by cathteng**

### CODE REVIEW

File: src/docs/product/accounts/quotas/spike-protection.mdx [hunk 1]
```diff
@@ -5,7 +5,9 @@ sidebar_order: 10
 description: "Learn how to manage your quota and make sure that your volume isn't consumed all at once with Spike Protection."
 ---
 
-A spike is a significant, temporary increase in your error event volume. Because Sentry bills based on the number of monthly events sent, spikes can quickly consume your quota for an entire month. Spike Protection guards against this by establishing a spike threshold for the average number of error events each of your projects sends and then dropping them after that threshold is reached.
+A spike is a significant, temporary increase in your event volume. Because Sentry bills based on the number of monthly events sent, spikes can quickly consume your quota for an entire month. Spike Protection guards against this by establishing a spike threshold for the average number of events each of your projects sends and then dropping them after that threshold is reached.
```
Comment (EricHasegawa) [inline src/docs/product/accounts/quotas/spike-protection.mdx:8]: nit: change to `... bills based on the number of events sent monthly`?

**Review by EricHasegawa — APPROVED** [review]
is this copy just written by us or does it go through docs as well?

**Comment by cathteng** [comment]
@EricHasegawa docs team will take a look too

File: src/docs/product/accounts/quotas/spike-protection.mdx [hunk 2]
```diff
@@ -5,7 +5,9 @@ sidebar_order: 10
...
+
+Spike protection currently applies to errors, transactions, and attachments.
```
Comment (vivianyentran) [inline src/docs/product/accounts/quotas/spike-protection.mdx:10]:
```suggestion
Spike Protection currently applies to errors, transactions, and attachments.
```

capitalize for consistency with rest of doc

**Review by vivianyentran — APPROVED** [review]
You might want to update error event --> event in line 30 as well.

**Review by vivianyentran — APPROVED** [review]
LGTM, thanks for updating!

**Pull request merged**